scripts/source/livecompare.js

```diff
@@ -19,7 +19,7 @@ const { exec, execSync } = require("child_process");
 const isAbsoluteUrl = require("is-absolute-url");
 
 const fileToMetadata = {};
-const basePath = path.resolve("temp_livecompare/docs/");
+const basePath = path.resolve("source/docs/");
```

Comment:
probably just make this a parameter for consistency with the CNP flow. Also the destination repo path, as right now this script assumes (line 50) that it'll be invoked from the root